packages/expo-image-loader/android/src/main/java/expo/modules/imageloader/ImageLoaderModule.kt

```diff
@@ -88,4 +90,12 @@ class ImageLoaderModule(val context: Context) : InternalModule, ImageLoader {
         }
       })
   }
+
+  private fun normalizeAssetsUrl(url: String): String {
+    var actualUrl = url
+    if (url.startsWith("asset:///")) {
+      actualUrl = "file:///android_asset/" + url.split("/").last()
```

Comment:
Are you sure converting it in that way is correct? I mean `asset:///` -> `file:///android_asset/`

Reply:
Yes, it is. There are issues in Glide, but most of them points to [this SO answer](https://stackoverflow.com/a/29987915), seems to be their design. However, undocumented. @sjchmiela is currently working with Glide, so he might be of help here

Reply:
Yeah, looking into Glide's code this seems to be the way to fetch assets, see:
- [`AssetUriResolver`](https://github.com/bumptech/glide/blob/d8278f43a94484f20822170b71ac7b66269a2aa6/library/src/main/java/com/bumptech/glide/load/model/AssetUriLoader.java)
- [test for `file:///android_asset/`](https://github.com/bumptech/glide/blob/55a475806c2014c54ed18ceb4cffd1f607a818d4/library/test/src/test/java/com/bumptech/glide/load/model/AssetUriLoaderTest.java#L38-L48)

Reply:
Another way to solve this would be to implement our own resolver that would `handle` `asset:///` URIs (unfortunately from my experience properly adding extensions to Glide's flow requires some hassle, so it may not be worth it).